client/src/components/Chart copy.js

```diff
@@ -0,0 +1,299 @@
+import { format } from "d3-format";
+import { timeFormat } from "d3-time-format";
+
+import React from "react";
+import PropTypes from "prop-types";
```

Comment:
Is this file even used? I don't see it anywhere.  If not then you should delete it.

Reply:
deleted